sapi/litespeed/lsapi_main.c

```diff
@@ -466,7 +466,7 @@ static sapi_module_struct lsapi_sapi_module =
 };
 /* }}} */
 
-static int init_request_info( TSRMLS_D )
+static void init_request_info( TSRMLS_D )
```

Comment:
Thank you. I forgot that :(
I'll close this PR and merge this to our repository from my local git.php.net checkout if you don't see any other issues with this PR.